lib/mix/tasks/write_stop_place_ids.ex

```diff
@@ -0,0 +1,159 @@
+defmodule Mix.Tasks.WriteStopPlaceIds do
+  @moduledoc """
+  Creates a `Dotcom.StopPlaceIds` module with a single function `stop_place_id/1`.
+  The function is a simple one to one mapping of stop ids to Google place ids.
+  """
+
+  @shortdoc "Create the `Dotcom.StopPlaceIds` module."
+
+  use Mix.Task
+
+  @file_path "lib/dotcom/stop_place_ids.ex"
+  @google_api_key System.get_env("GOOGLE_API_KEY", "")
+  @google_url "https://places.googleapis.com/v1/places:searchNearby"
+  @included_types [
+    "bus_station",
+    "bus_stop",
+    "ferry_terminal",
+    "subway_station",
+    "train_station",
+    "transit_station"
+  ]
+
+  @impl Mix.Task
+  def run(_) do
+    :ok = write_header()
+
+    :ok = write_place_ids()
+
+    :ok = write_footer()
+  end
+
+  # Given a latitude and longitude, construct a request body for Google.
+  # We don't use the name of the location.
+  # Rather, we use the latitude and longitude, restrict the types of places returned,
+  # order them by distance, and take the first.
+  # This means that accuracy is based on the stop latitude and longitude accuracy
+  # which should be more accurate than trying to match names.
```

Comment:
question(non-blocking): Would adding the name here help (hence attempting matching on lat/long _and_ name), or would that make the success rate worse?

Reply:
I think it makes it worse for some places where a) the name is vague, and/or b) the name is very close to another name.